src/record/import/repositories/error.ts

```diff
@@ -0,0 +1,30 @@
+import { ParserError } from "./parsers/error";
+
+export class RepositoryError extends Error {
+  private readonly cause: unknown;
+
+  constructor(cause: unknown) {
+    const message =
+      "An error occurred while loading records from the data source";
+    super(message);
+
+    this.name = "RepositoryError";
+    this.message = message;
+    this.cause = cause;
+
+    // https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#extending-built-ins-like-error-array-and-map-may-no-longer-work
+    // Set the prototype explicitly.
+    Object.setPrototypeOf(this, RepositoryError.prototype);
+  }
+
+  toString(): string {
+    let errorMessage = "";
+    errorMessage += this.message + "\n";
+    if (this.cause instanceof ParserError) {
```

Comment:
@ueokande 
I had assumed that `ParserError` should be wrapped in `RepositoryError`, because `Parser` is one of the details in Repository (Gateway). But I have forgotten to do it.
Which is better to remove this line or wrap errors from `Parser` into `RepositoryError`?

Reply:
At this time, keep the current implementation.